majilove/models.py

```diff
@@ -211,12 +223,12 @@ def max_max_level(self):
         return self.rarity_max_levels[1] if self.combinable else self.rarity_max_levels
 
     COLOR_CHOICES = [
-        ('star', _('Star')), # Yellow
-        ('shine', _('Shine')), # Red
-        ('dream', _('Dream')), # Blue
+        ('star', _('Star (Yellow)')), # Yellow
+        ('shine', _('Shine (Red)')), # Red
+        ('dream', _('Dream (Blue)')), # Blue
     ]
 
-    i_color = models.PositiveIntegerField(_('Color'), choices=i_choices(COLOR_CHOICES), db_index=True)
+    i_color = models.PositiveIntegerField(_('Attribute (Color)'), choices=i_choices(COLOR_CHOICES), db_index=True)
 
```

Comment:
Changed Colors to display as Attributes. Can be reverted if database just stores them as their color names.